_specs_/rest/paths/forex/historicTrades.yml

```
@@ -51,28 +51,38 @@ responses:
             - $ref: '../../schemas/index.yml#/ForexHistoricTrades'
             
         example:
-          day: 2020-10-14
-          map:
-            a: ask
-            b: bid
-            t: timestamp
-          msLatency: 3
-          pair: AUD/USD
-          status: success
-          ticks:
-            - b: 0.7177
-              a: 0.7182
-              x: 48
-              t: 1601596801000
-            - b: 0.71804
-              a: 0.71816
-              x: 48
-              t: 1601596802000
-            - b: 0.7178
-              a: 0.7183
-              x: 48
-              t: 1601596803000
-          type: forex
+          {
+            "day": "2020-10-14",
+            "map": {
+              "a": "ask",
+              "b": "bid",
```

This should be changed right?

```suggestion
              "ap": "ask",
              "bp": "bid",
```

---

Nice catch, letters confuse me.